components/link/link-mixin.js

```diff
@@ -57,14 +52,19 @@ export const LinkMixin = superclass => class extends LocalizeCoreElement(supercl
 		`];
 	}
 
-	_render(inner, { rel = undefined, linkClasses = {}, tabindex = undefined } = {}) {
+	getNewWindowDescription(label) {
+		return label && this.target === '_blank' ? this.localize('components.link.open-in-new-window') : undefined;
```

Comment:
What's the reason for the `label` check? ... wouldn't we still want this description if they are providing content in `inner`?